docs/api_docs/python.rst

```diff
@@ -0,0 +1,170 @@
+2. Create a variable ``client_files`` with the path to your additional client json files
+3. Create a variable module as a ``Vertiq4006`` module with module ID 42, speed firmware, and extra clients path client_files
+
+.. code-block:: python
+
+    import iqmotion as iq
+
+    # Module Communication
+    com = iq.SerialCommunicator("COM3", baudrate=115200)
+
+    # Clients to Load
+    client_files = "clients/"
+
+    # Using the Vertiq4006 with additional custom client files
+    module = iq.Vertiq4006(com, module_idn=42, firmware="speed", clients_path=client_files)
+
+At this point, you can communicate with, configure, and control your connected module through its available clients.
+
+Interacting with Clients and Endpoints
+-------------------------------------------------
+As mentioned in :ref:`getting_started_with_apis`, all Vertiq clients contain endpoints that can accept get, set, and save commands. This section discusses 
+how to perform gets, sets, and saves through the Python API.
+
+Before moving forward, please familiarize yourself with the clients available for your module's family and firmware style. You can find this information on your 
+module's family page in the Supported IQUART Clients section. For these examples, we will continue to use the ``Vertiq4006`` object created above.
+
+.. note::
+    In all instances, the value of ``client_entry`` is a value specified by the *Short Name* column of the associated IQUART client table.
+
+Get
+^^^^^^^^^^
+All Python get commands have the format ``module.get("client", "client_entry")``. The get function returns the value of a single client entry returned by the module through IQUART.
+
+Suppose we want to monitor the voltage read at the module's input. We can do this through the volts entry of the :ref:`Power Monitor <power_monitor>` client.
+
+.. image:: ../_static/api_pics/volts_entry.png
+
+In order to get and view the value of the volts parameter, we can add the following to our example
+
+.. code-block:: python
+
+    print(module.get("power_monitor", "volts"))
+
+You can also treat the returned value as a normal parameter, and store it in a variable.
+
+Get All
+^^^^^^^^^^
+All Python get_all commands have the format ``module.get_all("client”)``. The get_all function returns the value of all client entries returned by the module through IQUART.
+
+Suppose we want to see the current state of all parameters in the :ref:`System Control <system_control>` client.
+
+To do so, we can do the following:
+
+.. code-block:: python
+
+    print(module.get_all("system_control"))
+
+Again, you can also treat the returned value as a normal parameter, and store it in a variable. In this case the data is stored as an array.
+
+Set
+^^^^^^^^^^
+All Python set commands have the format ``module.set("client", "client_entry", value)``. The set function changes the value of the target ``client_entry`` to ``value``. 
```

Comment:
This is actually not quite true. You can do module.set("client", "client_entry") with no value for some entries I believe, like module.set("system_control", "reboot_program"), and it will still do what you want. May be worth calling out the existence of that type of entry in your set section